CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveAsAction.java

```diff
@@ -181,10 +182,14 @@ public boolean isEnabled() {
         return super.isEnabled();
     }
 
+    public boolean isSaved() {
+        return isSaved;
+    } 
+
     @Override
     public void actionPerformed(final ActionEvent e) {
         refreshListeners();
-        Collection<? extends SaveAsCapable> inst = lkpInfo.allInstances();
+        Collection<? extends SaveAsCapable> inst = lkpInfo.allInstances();        
```

Comment:
Looks like you've added some random whitespace to the end of this line. It's not a big enough deal to worry about fixing now, but there is an 'on save' option in Netbeans which will prevent this sort of thing next time.

Reply:
That's a good tip. Thanks. Removed this.